WcaOnRails/config/application.rb

```diff
@@ -10,7 +10,9 @@
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
-ENVied.require(*ENV['ENVIED_GROUPS'] || Rails.groups)
+
+Dotenv::Railtie.load
```

Comment:
I think that shouldn't be necessary given that we require `rails-now` as part of the bundler definition for `dotenv`. If it doesn't work without this line, then please investigate with the help of https://github.com/bkeepers/dotenv#note-on-load-order

Reply:
ok I will try that